### PR TITLE
BUG: Don't display items containing multiple files in item_preview plugin

### PR DESCRIPTION
**Pull request by MayeulChassagnard**
Since an item containing multiple files is going to be download as a zip file,
and the item_preview do not support a display inline of a zip file, it produces
a display error.

Then, one solution is to check all items and remove the ones containing
multiple files from `supportedItems`.

Cons:
* Do not preview:
  - supported item containing multiple supported files
  - item named `example` containing a supported file (indeed, for now,
    an item is defined as supported if its name contain .png at the end
    for instance)
* Error still shown:
  - if an item considered as supported (with .png at the end of its name for
    instance) and the file inside is not supported (a .txt file for example)

### CODE REVIEW

**Comment by MayeulChassagnard** [comment]
This PR fix the issue #1649

**Comment by jcfr** [comment]
Thanks for the PR.

Before we move forward with the review process, consider fixing the failing test. 

Assuming girder sources are in `~/Projects/girder`, you can do the following to run the tests:

```bash
$ mkdir ~/Projects/girder-build
$ cd ~/Projects/girder-build
$ workon girder_env # This assumes you have a virtual env named girder_env
$ cmake ../girder
$ ctest -R item_previews
Test project /home/jcfr/Projects/girder-build
    Start 128: eslint_item_previews
1/2 Test #128: eslint_item_previews .............   Passed    0.59 sec
    Start 129: puglint_item_previews
2/2 Test #129: puglint_item_previews ............   Passed    0.13 sec
```

File: plugins/item_previews/web_client/views/ItemPreviewWidget.js [hunk 1]
```diff
@@ -53,9 +54,38 @@ var ItemPreviewWidget = View.extend({
         return /\.(jpg|jpeg|png|gif)$/i.test(item.get('name'));
     },
 
+    _isSingleFileInItem: function (item) {
```
Comment (jcfr) [inline plugins/item_previews/web_client/views/ItemPreviewWidget.js]: `_isSingleFileItem`

That said this is more explicit: `_isMultifileFileItem`

Comment (MayeulChassagnard) [inline plugins/item_previews/web_client/views/ItemPreviewWidget.js]: I used this name previously indeed but what if there is no files in an item? We will have to return the same answer that if there was a single item, which could be annoying I think

Comment (jcfr) [inline plugins/item_previews/web_client/views/ItemPreviewWidget.js]: Makes sense. Then, I would suggest a function name like `_isSingleFileItem`

**Comment by zachmullen** [comment]
Let's reimplement this once we cache the `nFiles` value on items, which will save us from doing O(n) XHRs on the list view.

**Pull request closed without merging**